WebRoot/js/zimbraAdmin/servers/controller/ZaServerController.js

```diff
@@ -265,26 +265,43 @@ function (params) {
 						}
 					}
 				} else {
-					//include
+		    //include
                     validStr = chunks[i];
-                    try {
-                        cidrData = ZaIPUtil.isValidCIDR(validStr);
-                    } catch (ex) {
-                        throw new AjxException(AjxMessageFormat.format(ZaMsg.ERROR_NOT_CIDR,[validStr]),AjxException.INVALID_PARAM,"ZaServerController.prototype.validateMyNetworks");
-                    }
-                    masks.push(cidrData);
 
-					for(var j=(IFCounter-1);j>=0;j--) {
+                    // bug ZCS-1549:  allow non-CIDR inet addresses
+                    var j=0;
```

Comment:
can we have spaces around operators (=, < etc) in general, that would be better for readability here.